crates/cheatnet/src/constants.rs

```diff
@@ -45,18 +45,46 @@ pub const TEST_FAULTY_ACCOUNT_CONTRACT_CLASS_HASH: &str = "0x113";
 pub const SECURITY_TEST_CLASS_HASH: &str = "0x114";
 pub const TEST_ERC20_CONTRACT_CLASS_HASH: &str = "0x1010";
 
+pub const STEP_RESOURCE_COST: f64 = 0.01_f64;
+
+// HOW TO FIND:
+// 1. https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#general_case
+// 2. src/starkware/cairo/lang/instances.py::starknet_with_keccak_instance
```

Comment:
Should we check them every time we update the compiler version?

Reply:
Basically yes xD Unless you wanna use python bindings for cairo lang, which blockifier uses